functions_framework_tool/lib/src/cli/context.dart

```diff
@@ -0,0 +1,42 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'dart:io' as io;
+
+import 'package:stagehand/stagehand.dart';
+
+import 'console.dart';
+
+class Context {
```

Comment:
As in "Each Cloud Function runs in its own isolated secure execution context" or as in Go's `context` type? I'll rename to `CommandContext` to avoid confusion.

Reply:
Renamed to `CommandContext` and moved to `command.dart`.